deploy/helm/kubecf/values.yaml

```diff
@@ -45,12 +45,31 @@ releases:
 sizing:
   HA: false
 
-# Service is only valid to set an external endpoint for the router if features.ingress.enabled is
-# false.
-service:
-  type: LoadBalancer
-  externalIPs: []
-  clusterIP: ~
+# Service is only valid to set a external endpoints for the instance groups if
+# features.ingress.enabled is false.
+services:
+  router:
+    type: LoadBalancer
+    externalIPs: []
+    clusterIP: ~
+    ports:
+    - name: http
+      protocol: TCP
+      port: 80
+      targetPort: 8000
+    - name: https
+      protocol: TCP
+      port: 443
+      targetPort: 443
```

Comment:
I don't like this… the admin (the people doing the deployment) wouldn't ever want to change this, since the only thing they could do is to fiddle with it and end up with a broken deployment.

We'll need to do something different here; unfortunately I have no idea what that is yet.  I think I'd even prefer a duplicated (but hard-coded) section over this. (But we could probably at least use a local template in that case.)

Reply:
I left a duplicated service for now.